api/api/views/viewsets/vulns.py

```diff
@@ -16,3 +18,19 @@ class NotesViewset(viewsets.ModelViewSet): # pylint: disable=too-many-ancestors
     permission_classes = [permissions.IsAuthenticated & IsAdmin | IsOwner] # FIXME(adina): add isPartOfTheTeam
     authentication_classes = [TokenAuthentication]
     serializer_class = NotesSerializer
+
+class VulnTypeViewset(viewsets.ModelViewSet):
+    """
+        Crud to add a new vulnerability type. Shouldn't happen often, but still.
+    """
+    queryset = VulnType.objects.all()
+    permissions = [permissions.IsAuthenticated]
+    authentication_classes = [TokenAuthentication]
+    serializer_class = VulnTypeSerializer
+
+
+class VulnerabilityViewset(viewsets.ModelViewSet):
+    queryset = Vulnerability.objects.all()
+    permissions = [permissions.IsAuthenticated & IsAdmin | IsOwner] # FIXME(adina): add is PartOfTheTeam
```

Comment:
Managers should not be able to add a new Vulnerability, you can just use `[permissions.IsAuthenticated & IsOwner]` here i think